test/e2e/apps/disruption.go

```diff
@@ -58,7 +58,13 @@ var _ = SIGDescribe("DisruptionController", func() {
 		createPDBMinAvailableOrDie(cs, ns, intstr.FromString("1%"))
 	})
 
-	ginkgo.It("should update PodDisruptionBudget status", func() {
+	/*
+	   Release : v1.17
```

Comment:
need to bump this right?

Reply:
Yeah, I have updated release version to 1.21.